README.md

```diff
@@ -108,7 +174,7 @@ This will also send metrics and traces to Loki as an unintended side effect.
 Add the following command line parameter:
 
 ```shell
--Dgrafana.otlp.debug.logging=true
+export GRAFANA_OTLP_DEBUG_LOGGING=true
```

Comment:
That's one last spot where the `OTLP_` part is used, but I guess this is fine?

Reply:
yes, that's intentional - because it's about otlp data